airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -56,6 +57,7 @@ class EmrAddStepsOperator(BaseOperator):
     :param wait_for_completion: If True, the operator will wait for all the steps to be completed.
     :param execution_role_arn: The ARN of the runtime role for a step on the cluster.
     :param do_xcom_push: if True, job_flow_id is pushed to XCom with key job_flow_id.
+    :param deferrable: if True, the operator will run in deferrable mode.
```

Comment:
maybe add something saying that this implies waiting for completion (I know it sets it to False, but from the user perspective, it's going to effectively wait the the completion)

Reply:
That's a good idea. This is true for all deferrable mode operators, so I'll update the doc strings for the other deferrable PRs I've got.